src/main/java/Botty.java

```diff
@@ -0,0 +1,69 @@
+package main.java;
+
+import main.java.command.Command;
+import main.java.exception.EmptyChoiceException;
+import main.java.exception.EmptyTodoException;
+import main.java.exception.UnknownCommandException;
+import main.java.parser.Parser;
+import main.java.storage.Storage;
+import main.java.tasklist.TaskList;
+import main.java.ui.Ui;
+import java.util.Scanner;
+
+/**
+ * The Botty class represents the main class responsible for interacting with the user and managing tasks.
+ * It initializes the user interface, task list, parser, and storage components.
+ * The class reads user input, processes commands, and handles task operations.
+ */
+public class Botty {
+
+    private TaskList taskList;
+    private Ui ui;
+    private Parser parser;
+    private Storage storage;
+
+    /**
+     * Constructs a new `Botty` instance, initializing user interface, task list, parser, and storage components.
+     * The constructor also loads the task list from storage.
+     */
+    public Botty() {
+        this.ui = new Ui();
+        this.taskList = new TaskList();
+        this.parser = new Parser();
+        this.storage = new Storage("./storage.txt");
+        this.storage.loadTaskList(this.taskList);
+    }
+    public static void main(String[] args) {
```

Comment:
Perhaps there could be a blank line before the main method to improve code readability.